Puccinia triticum.md upload

RF216 opened the pull request.

RF216 committed:
Puccinia triticum.md upload

ababaian commented on virus/img/PtV_images/.DS_Store:
remove all dot files.

ababaian commented on virus/img/Puccinia_triticum/Figure_1.0.png:
the file-path for your images should be `virus/img/<Genus_Species>/<file>` don't abbreviate the foldername and fix all references to those images in your report.

ababaian commented on virus/Puccinia_triticum.md:
```diff
@@ -0,0 +1,109 @@
+# {Q1A Paper Title}
```
For anything in squiggly brackets, you need to fill it out with the appropriate information and delete the squiggly brackets.

Your report title goes here.

ababaian commented on virus/Puccinia_triticum.md:
```diff
@@ -0,0 +1,109 @@
+# {Q1A Paper Title}
+
+written by: [Ryan Fu](https://github.com/yourname%20*optional%20link*)
```
either link your github account or remove the link

ababaian commented on virus/Puccinia_triticum.md:
```diff
@@ -0,0 +1,109 @@
+
+## Results
+
+### Ecology subsection A...
```
Don't use generic titles, make them specific to your report.

ababaian commented on virus/Puccinia_triticum.md:
```diff
@@ -0,0 +1,109 @@
+
+The most notable hit is the putative Rdrp with accession WBC51218.1, referring to a paper describing a novel ambigrammatic mycovirus, PsV5 that allegedly works hand in hand with wheat rust fungus to promote infection (Zhang et al). This hit, especially with a %identity of 70% provides evidence for our primary hypothesis. This paper proposes a novel narnavirus PsV5 that contains an ambigrammatic s ...
+
+### Virus Genome {Q3}
```
Remove the brackets which reference a question

ababaian commented on virus/Puccinia_triticum.md:
```diff
@@ -0,0 +1,109 @@
+
+Analyzing the SRA runs obtained from serratus, containing our sequence, we can determine which species’ our virus most likely infects.
+
+![](img/PtV_images/Figure_1.0.png)
```
For all of your figures, add a short figure description in the markdown so that the Display text is shown if the image is not rendered.

ababaian reviewed (changes requested):
Few changes required to close the PR

RF216 committed:
Update title of Puccinia triticum document

RF216 committed:
Fix formatting issues in Puccinia triticum markdown

RF216 committed:
Fix author link and improve text formatting

Updated author link and corrected minor text formatting.

RF216 committed:
Revise hypothesis and update Virus Genome section

Adjusted primary hypothesis regarding the cooperative role of the novel virus and PsV5 in promoting wheat rust infection. Updated section heading for clarity.

RF216 committed:
Revise figure captions and descriptions in document

Updated figure captions for clarity and consistency throughout the document. Enhanced descriptions for images related to virus analysis and ORF findings.

RF216 committed:
Revise Puccinia triticum report and results section

Updated the description of Puccinia triticum and its correlation with wheat rust disease. Improved clarity in the results section regarding the ecology of the virus.

RF216 committed:
Delete virus/img/PtV_images/.DS_Store

RF216 committed:
Rename Figure_1.0.png to Figure_1.0.png

RF216 committed:
Rename Figure_10.png to Figure_10.png

RF216 committed:
Rename Figure_11.png to Figure_11.png

RF216 committed:
Rename Figure_12.png to Figure_12.png

RF216 committed:
Rename Figure_2.0.png to Figure_2.0.png

RF216 committed:
Rename Figure_3.0.png to Figure_3.0.png

RF216 committed:
Rename Figure_4.0.png to Figure_4.0.png

RF216 committed:
Rename Figure_5.png to Figure_5.png

RF216 committed:
Rename Figure_6.png to Figure_6.png

RF216 committed:
Rename Figure_7.png to Figure_7.png

RF216 committed:
Rename Figure_8.png to Figure_8.png

RF216 committed:
Rename Figure_9.png to Figure_9.png

RF216 committed:
Edit image pathways

RF216 committed:
Create Puccinia_triticum_vid.md

RF216 committed:
Revise virus description and add abstract section

Updated video description and added abstract about Puccinia triticum virus. Removed placeholder comments and added presenter and project links.

RF216 committed:
Fix project link in Puccinia_triticum_desc.md

Updated project link to correct URL format.

RF216 committed:
Add files via upload

RF216 committed:
Rename virus/vid/Puccinia_triticum_vid_thumbnail.png to virus/vid/img/Puccinia_triticum_vid_thumbnail.png

RF216 committed:
Update video thumbnail path for Puccinia triticum

RF216 committed:
Rename virus/vid/img/Puccinia_triticum_vid_thumbnail.png to virus/vid/img/Puccinia_triticum/Puccinia_triticum_vid_thumbnail.png

RF216 committed:
Fix video thumbnail formatting in markdown

Updated video thumbnail syntax in markdown.

RF216 committed:
Update Puccinia_triticum_desc.md

RF216 committed:
Merge branch 'main' into main

RF216 committed:
Rename virus/vid/img/Puccinia_triticum/Puccinia_triticum_vid_thumbnail.png to virus/img/Puccinia_triticum/Puccinia_triticum_vid_thumbnail.png

RF216 committed:
Update Puccinia_triticum_desc.md

RF216 commented:
Fixed + Video Upload!

ababaian reviewed (approved):
gj!
